crates/holochain_state/src/mutations.rs

```diff
@@ -404,12 +404,21 @@ pub fn insert_wasm(txn: &mut Transaction, wasm: DnaWasmHashed) -> StateMutationR
 }
 
 /// Insert a [`DnaDef`] into the database.
-pub fn insert_dna_def(txn: &mut Transaction, dna_def: &DnaDefHashed) -> StateMutationResult<()> {
-    let hash = dna_def.as_hash();
-    let dna_def = dna_def.as_content();
-    sql_insert!(txn, DnaDef, {
```

Comment:
We had a second layer of protection here against accidentally updating coordinator zomes :stuck_out_tongue: . The SQL statement was only an `INSERT` and would never actually replace an existing `DnaDef` in the database if an entry for the given dna hash already existed.

Reply:
But didn't report that.